waveorder/visuals/napari_visuals.py

```diff
@@ -0,0 +1,75 @@
+from waveorder.visuals.utils import complex_tensor_to_rgb
+
+import numpy as np
+import torch
+
+
+def add_transfer_function_to_viewer(
+    viewer,  # napari viewer
```

Comment:
Type hint can be guarded in `typing.TYPE_CHECKING` or in strings.

Reply:
Thank you!

Reply:
Hmmm I couldn't figure out how to get `TYPE_CHECKING` to work...even after comparing with what you've been doing in recOrder. 

Changed to strings.

Reply:
That needs `from __future__ import annotations` to work. String is equivalent though.